.gitignore

```diff
@@ -5,4 +5,5 @@ package-lock.json
 node_modules
 /android/build
 .idea/
-yarn.lock
\ No newline at end of file
+yarn.lock
+.env
```

Comment:
Let's rename `.env` to `.env-test`. In this case we will know that it's used for test. Also add those env vars description to readme file or create .env-example .
Additionally we need to update documentation. You could find it on `documentation/index.html` folder